README.md

```diff
@@ -255,6 +256,20 @@ console.log(
 );
 ```
 
+### Find render-blocking resources
+
+List all resources that are blocking rendering.
+
+```js
+function findRenderBlockingResources() {
+  return window.performance.getEntriesByType('resource')
+    .filter(({renderBlockingStatus}) =>
+        renderBlockingStatus === 'blocking');
+}
+
+console.log(findRenderBlockingResources());
```

Comment:
Thanks a lot @stefanjudis 👏

What do you think about listing the relevant data as a table to improve the reading?
How about listing the relevant data in a table to improve reading?

```suggestion
function findRenderBlockingResources() {
  return window.performance.getEntriesByType('resource')
    .filter(({renderBlockingStatus}) =>
        renderBlockingStatus === 'blocking')
    .map(({startTime, duration, responseEnd, name, initiatorType}) => [startTime, duration, responseEnd, name, initiatorType]);
}
console.table(findRenderBlockingResources());
```